### PR TITLE
Allow switching prisons

### PR DESCRIPTION
**Pull request by rossjones**
If a POM has acess to more than one prison, then this PR allows them to
switch between them.

### CODE REVIEW

**Comment by rossjones** [comment]
Should probably remove the old ‘switcher’ from the header before review.

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -20,8 +20,15 @@ def active_caseload
...
+  def update_active_caseload(code)
+    session[:sso_data]['active_caseload'] = code
+  end
+
   def caseloads
-    sso_identity['caseloads'] if sso_identity.present?
+    return nil if sso_identity.blank?
+
+    caseloads = sso_identity['caseloads']
+    caseloads.reject { |c| c == 'NWEB' }
```
Comment (TobyRet) [inline app/controllers/application_controller.rb]: How come NWEB get's returned in caseloads?

File: app/views/search/search.html.erb [hunk 2]
```diff
@@ -1,3 +1,6 @@
+<% content_for :switcher do %>
```
Comment (TobyRet) [inline app/views/search/search.html.erb]: I wonder if 
```
<% content_for :switcher do %>
  <%= render '/layouts/prison_switcher' %>
<% end %>
```

can be moved into layouts/application.html.erb  - then it's only defined in one place

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (KPobeeNorris) [reply app/controllers/application_controller.rb]: Because NWEB is assigned under the caseload section in NOMIS, the same screen where prisons are added

**Pull request merged**